clients/privacy-center/cypress/e2e/consent.cy.ts

```diff
@@ -133,7 +150,7 @@ describe("Consent settings", () => {
       });
 
       // The cookie should also have been updated.
-      cy.getCookie(CONSENT_COOKIE_NAME).then((cookie) => {
+      cy.getCookie(CONSENT_COOKIE_NAME).should((cookie) => {
```

Comment:
There have been [test flakes](https://github.com/ethyca/fides/actions/runs/4179920293/jobs/7240389906) around this block. I wasn't able to reproduce the failure locally (tried running cypress in a loop in my shell) but clearly CI is doing something too fast or too slow for the cookie to have been updated. I have a theory that [should()](https://docs.cypress.io/api/commands/should) will get rid of the flake because cypress will intelligently wait for the chain of assertions to pass, unlike `then` which just immediately hands off the value.